x-pack/plugins/ml/public/application/data_frame_analytics/common/get_index_data.ts

```diff
@@ -63,7 +63,13 @@ export const getIndexData = async (
 
       if (!options.didCancel) {
         setRowCount(resp.hits.total.value);
-        setTableItems(resp.hits.hits.map((d) => getProcessedFields(d.fields)));
+        setTableItems(
```

Comment:
Guess this is not needed now that https://github.com/elastic/kibana/pull/82334 is merged?